storybook/src/Storybook/InspectionItemsList.qml

```diff
@@ -11,7 +11,9 @@ ListView {
 
     delegate: Text {
         width: ListView.view.width
-        height: 30
+        height: implicitHeight * 1.5
+
+        font.pixelSize: ApplicationWindow.window.font.pixelSize - 1
```

Comment:
How can the `window` be null here? 🤔 

Sometimes when selecting an item to inspect there will be this error:
InspectionItemsList.qml:16: TypeError: Cannot read property 'font' of null

Go to ProfileDialogView -> Inspect -> Click on `MockNickname hasn't joined any communities yet`

Reply:
Thank you for spotting that! Sometimes it's better to keep simple things simple ;)